docs/install/common-problems.md

```diff
@@ -41,7 +41,10 @@ If that is not an option, then the only remote access we recommend is using some
 
 One common error some users are seeing manifests itself just after selecting an EDEX server to connect to.  The following error dialogs may show up:
 
+- *Error purging logs*  
 ![Error purging logs. Reason: Error occurred during purge and rotate](../images/errorPurgingLogs.png)
+
+- *Error instantiating workbench*  
```

Comment:
Want to add  ": null" or just leave it?

Reply:
I thought about it but couldn't decide... what do you think?

Reply:
I think for that one it should have it

Reply:
Okay just added that!